qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -134,19 +134,60 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
             }
             hasNetworkViolation = results.any { isNetworkViolation(it) }
             hasProcessViolation = results.any { isProcessViolation(it) }
```

Comment:
How do we know for sure that there will be network or process violations? Does this test create deployments that trigger these kind of violations?

Reply:
Yes, see [lines 346, ff](https://github.com/stackrox/stackrox/pull/11848/files#diff-b5d8e8c78f856dbec06efbf9fdf975540853d6691648c5071a853559e9e47a5bR346-R352). The test creates deployments that result in process and network violations.